mapper/src/test/scala/org/scalarelational/mapper/AsyncSpec.scala

```diff
@@ -68,5 +68,7 @@ object AsyncDatastore extends H2Datastore(mode = H2Memory("async_test")) {
     val id = column[Option[Int], Int]("id", PrimaryKey, AutoIncrement)
     val name = column[String]("name")
     val age = column[Int]("age")
+
+    override def query = q.to[AsyncUser]
```

Comment:
`def query` is part of `MappedTable` so wouldn't be required, although there's nothing stopping you from introducing the method yourself it just wouldn't override anything.